cmake/jerry.cmake

```diff
@@ -83,6 +83,13 @@ if("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
   list(APPEND DEPS_LIB_JERRY_ARGS -DFEATURE_ERROR_MESSAGES=ON)
 endif()
 
+# Nuttx is not using the default port implementation of JerryScript
+if("${TARGET_OS}" MATCHES "NUTTX")
+  list(APPEND DEPS_LIB_JERRY_ARGS -DJERRY_PORT_DEFAULT=OFF)
```

Comment:
@LaszloLango I am getting several errors during build for tizenrt-artik05x. We will make another PR for tizen-artik05x. Let land this PR separately.